test_requirements.txt

```diff
@@ -1,7 +1,7 @@
-flake8==3.2.1
+flake8==3.4.1
```

Comment:
Why can't we have these as just...
flake8
mock
six
pytest
pytest-cov
pytest-html